oracle/rpc/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+pub use self::gen_client::Client as OracleClient;
+use codec::Codec;
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+pub use orml_oracle_rpc_runtime_api::OracleApi as OracleRuntimeApi;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+use std::sync::Arc;
+
+#[rpc]
+pub trait OracleApi<BlockHash, Key, Value> {
+	#[rpc(name = "oracle_getNoOp")]
```

Comment:
```suggestion
	#[rpc(name = "orml_oracle_getValue")]
```

why call it getNoOp? RPC cannot have side effects on on-chain storage. I think getValue is fine.